diagrams/README.md

```diff
@@ -0,0 +1,7 @@
+Job execution path under the simulator:
+
+![simulated execution](./simulated_execution.svg)
+
+Job execution path under the non-simulated, actual version of Flux:
+
+![actual execution](./actual_execution.svg)
```

Comment:
Looks great. One thing: we haven't decided whether the resource module will pass the R to the scheduler control loop module or directly to KVS.

Reply:
That is a good point. I'll make a note in the README text about this.  I'll also add a note about anything in parentheses is a request/event that is still TBD.

Reply:
Sounds good. Another decision we will have to make is whether the sched control loop module will pass the jobspec string directly to the resource module or send only handle so that resource can find the jobspec from the KVS. My preference is the latter. This detail doesn't need to go to the diagram though.